S3TransferUtilitySample/build.gradle

```diff
@@ -34,7 +36,7 @@ android {
     defaultConfig {
         versionCode 1
         versionName "1.0.0"
-        minSdkVersion 19
+        minSdkVersion 23
```

Comment:
Add a comment saying why we bumped to 23?
// aws-android-sdk-mobile-client depends on aws-android-sdk-auth-ui which requires a minSdkVersion of 23

Reply:
Done!